completion/tracking.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""
+Tracking and analytics events for block completions.
+"""
+
+from django.conf import settings
+
+from eventtracking import tracker
+
+from openedx.core.djangoapps.site_configuration import helpers
+
+
+TRACKER_EVENT_NAME_FORMAT = u'edx.completion.{block_type}.{event_type}'
+
+
+def _is_trackable_block_type(block_type):
+    """
+    Checks settings to see if we want to track this block type.
+    We use a black list and assume tracking of all, otherwise.
+    """
+    return block_type not in helpers.get_value(
+        'COMPLETION_TRACKED_BLOCK_TYPE_BLACKLIST',
+        settings.COMPLETION_TRACKED_BLOCK_TYPE_BLACKLIST)
+
+
+def track_completion_event(blockcompletion, completion_percent, is_new):
+    """
+    Logic for emitting or skipping a tracking event for a completed block.
+
+    """
+    if helpers.get_value(
+        'COMPLETION_ENABLE_EVENTTRACKING', settings.COMPLETION_ENABLE_EVENTTRACKING) is not True:
+        return
```

Comment:
Removing `is not True` to make it a bit more Pythonic. Not sure about the line length though.

```suggestion
    if not helpers.get_value('COMPLETION_ENABLE_EVENTTRACKING', settings.COMPLETION_ENABLE_EVENTTRACKING):
        return
```